java/main/com/google/privacy/differentialprivacy/AggregationState.java

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.privacy.differentialprivacy;
+
+public enum AggregationState {
+    DEFAULT(""),
+    SERIALIZED(" object has been already serialized."),
+    RESULT_RETURNED("'s noised result is already computed and returned.");
```

Comment:
I don't think we want this specific to error message constructions here.
Remove space before object in SERIALIZED and 
"'s " here, and make changes in the code.  I don't think we need to specify in Java wchich object's noise it was, since the exception is thrown and the caller class (the aggregation type) will appear in the logs anyway.

Reply:
Change it in Go too to have the class name in the error message of the class rather than the state.